src/main/java/uk/gov/mca/beacons/api/services/GetValidationErrorResponse.java

```diff
@@ -0,0 +1,8 @@
+package uk.gov.mca.beacons.api.services;
+
+import org.springframework.validation.Errors;
+import uk.gov.mca.beacons.api.dto.ErrorResponseDTO;
+
+public interface GetValidationErrorResponse {
+  ErrorResponseDTO fromBindingErrors(Errors errors);
+}
```

Comment:
This is in the `services/` directory.  Isn't it a mapper? 

"Here are some `Errors`, turn these into `ErrorResponseDTOs`" -- similar to our other `map POJO to DTO` classes.

Reply:
Agreed.

Have moved into the `mappers` package and have updated the name of the interface/class